opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/jdbc/JdbcAutoConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package io.opentracing.contrib.spring.cloud.jdbc;
+
+import io.opentracing.contrib.jdbc.TracingConnection;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Loads the integration with OpenTracing JDBC if it's included in the classpath.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+@Configuration
+@ConditionalOnClass(TracingConnection.class)
```

Comment:
I think this is not necessary or even wrong because this class is a compile dependency.

please also provide a way how to disable this auto-config